docs/sources/manage/notify/phone-calls-sms/index.md

```diff
@@ -78,8 +78,316 @@ There are no specific rate limits, but we reserve the right to stop sending SMS/
 
 To learn the phone number used by OnCall, make a test call from the “Phone Verification” tab.
 
-### Phone calls or SMS not working
+### SMS/Voice support by country
 
-There are instances where OnCall may not be able to make phone calls or send SMS to certain regions or specific phone numbers. We are working to resolve these issues.
-Please test your personal notification chain to ensure OnCall can notify you.
-We also suggest backing up phone calls and SMS with other notification methods such as the [Mobile app](ref:mobile-app).
+The following is a list of countries currently supported by Grafana OnCall.
+
+{{< admonition type="note" >}}
+Please be aware that due to limitations
+in our telecom provider’s service, some numbers within supported countries may occasionally be flagged as “high-risk” when
+verifying your phone number, thereby preventing you from being able to use that number to receive notifications.
+
+We recommend testing your notification chain to confirm that OnCall can reach you. For added reliability, consider backing
+up phone calls and SMS notifications with additional methods, such as the [Mobile app](ref:mobile-app).
```

Comment:
```suggestion
up phone calls and SMS notifications with additional methods, such as the [Mobile app](/docs/grafana-cloud/alerting-and-irm/irm/irm-mobile-app/push-notifications/).
```

Reply:
These mobile app docs will be hidden from the TOC soon so I changed this link to point to the new mobile app docs.